### PR TITLE
FAT-2650/C350422

### PR DESCRIPTION
**Pull request by re-jas**
[C350422: Check that user can create "Page" Item level request](https://issues.folio.org/browse/FAT-2650)
https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1463/allure/

### CODE REVIEW

File: cypress/e2e/title-level-request/user-can-create-page-Item-level-request.cy.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      }).then(() => {
+        TitleLevelRequests.changeTitleLevelRequestsStatus('allow');
+      });
+      cy.login(userData.username, userData.password);
+    });
+  });
+
+  after('Deleting created entities', () => {
+    cy.get('@requestId').then((id) => {
+      Requests.deleteRequestViaApi(id);
```
Comment (MaksimTerekhov) [inline cypress/e2e/title-level-request/user-can-create-page-Item-level-request.cy.js:120]: add pls deleting other entities
user, items, instance

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/title-level-request/user-can-create-page-Item-level-request.cy.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        userData.userId,
+        testData.userServicePoint.id
+      );
+      cy.loginAsAdmin({
+        path: SettingsMenu.circulationTitleLevelRequestsPath,
+        waiter: TitleLevelRequests.waitLoading,
+      }).then(() => {
+        TitleLevelRequests.changeTitleLevelRequestsStatus('allow');
+      });
+      cy.login(userData.username, userData.password);
```
Comment (IhorBohdan) [inline cypress/e2e/title-level-request/user-can-create-page-Item-level-request.cy.js:114]: You can use here like example cy.login(user.username, user.password, { path:TopMenu.ordersPath, waiter: Orders.waitLoading }); 
and remove first cy.visit from it

**Review by IhorBohdan — APPROVED** [review]

File: cypress/e2e/title-level-request/user-can-create-page-Item-level-request.cy.js (hunk 2 above)
Comment (re-jas) [reply cypress/e2e/title-level-request/user-can-create-page-Item-level-request.cy.js]: done

File: cypress/e2e/title-level-request/user-can-create-page-Item-level-request.cy.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+      ruleProps.r = requestPolicyBody.id;
+      addedCirculationRule = 't ' + testData.loanTypeId + ': i ' + ruleProps.i + ' l ' + ruleProps.l + ' r ' + ruleProps.r + ' o ' + ruleProps.o + ' n ' + ruleProps.n;
+      CirculationRules.addRuleViaApi(originalCirculationRules, ruleProps, 't ', testData.loanTypeId);
+    });
+
+    cy.createTempUser(
+      [
+        permissions.uiUsersfeefinesCRUD.gui,
+        permissions.uiUsersfeefinesView.gui,
+        permissions.loansRenew.gui,
```
Comment (penfriend) [inline cypress/e2e/title-level-request/user-can-create-page-Item-level-request.cy.js:103]: do you need this permission&

Comment (re-jas) [reply cypress/e2e/title-level-request/user-can-create-page-Item-level-request.cy.js]: removed

**Review by penfriend — APPROVED** [review]

**Pull request merged**